src/components/Contributors/Contributor.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @author Samy Nastuzzi <samy@nastuzzi.fr>
+ *
+ * @copyright Copyright (c) 2019, SiMDE-UTC
+ * @license GPL-3.0
+ */
+
+import React from 'react';
+import { View, Text, Image, Linking } from 'react-native';
+import LinkButton from '../LinkButton';
+import colors from '../../styles/colors';
+
+export default function Contributor({
+	name,
+	subname,
+	description,
+	picture,
+	url,
+	backgroundColor,
+	roundedBottom,
+}) {
+	return (
+		<LinkButton
```

Comment:
It looks weird with a shadow on iOS. You should add a prop in LinkButton to disable it.